Summer at HF ☀️ (#159)
huggingface/blog: summer-at-huggingface.md

---
title: "Summer at Hugging Face"
thumbnail: /blog/assets/27_summer_at_huggingface/thumbnail.png

---

# Summer At Hugging Face

<div class="blog-metadata">

>     <small>Published July 13, 2021.</small>

Reviewer: ```suggestion
    <small>Published September 22, 2021.</small>
```

    <a target="_blank" class="btn no-underline text-sm mb-5 font-sans" href="https://github.com/huggingface/blog/blob/master/summer-at-huggingface.md">
        Update on GitHub
    </a>
</div>

<div class="author-card">
    <a href="/huggingface">

>         <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1583856921041-5dd96eb166059660ed1ee413.png?w=200&h=200&f=face" title="Gravatar">

Reviewer: Why don't you add yourself an the author @osanseviero ?

Author: A lot of people in the team contributed to individual sections, I didn't write it all. Having me as author would not be right, and given this is a post about everything we've worked on, I liked the idea of having HF as the author.

        <div class="bfc">
            <code>huggingface</code>
            <span class="fullname">Hugging Face</span>
        </div>
    </a>
</div>

Summer is almost over but the last few months have been busy at Hugging Face. From new features in the Hub to research and Open Source development, our team has been working hard to empower the community through open and collaborative technology. 

In this blog post you'll read about everything that has happened at Hugging Face in June, July and August!

> ![Summet At Hugging Face](assets/27_summer_at_huggingface/summer_intro.gif)

Reviewer 2: ```suggestion
![Summer At Hugging Face](assets/27_summer_at_huggingface/summer_intro.gif)
```

> This post covers a wide range of areas our team has been working on, don't hesitate to skip to the parts that interest you most 🤗

Reviewer 3: ```suggestion
This post covers a wide range of areas our team has been working on, so don't hesitate to skip to the parts that interest you most 🤗
```

> This post covers a wide range of areas our team has been working on, so don't hesitate to skip to the parts that interest you most 🤗

Reviewer 4: ```suggestion
This post covers a wide range of areas our team has been working on, so don't hesitate to skip to the parts that interest you the most 🤗
```

1. [New Features](#new-features)
2. [Community](#community)
3. [Open Source](#open-source)
4. [Solutions](#solutions)
5. [Research](#research)

## New Features

In the last few months, the Hub went from 10,000 public model repositories to over 16,000 models! Kudos to our community for sharing so many amazing models with the world. And beyond the numbers, we have a ton of cool new features to share with you!

### Spaces Beta ([hf.co/spaces](/spaces)) 

> Spaces is a simple, free solution to host Machine Learning demo applications directly on your user profile or your organization [hf.co](http://hf.co/) profile. We support two awesome SDKs that let you build cool apps easily in Python: [Gradio](https://gradio.app/) and [Streamlit](https://streamlit.io/). In a matter of minutes you can deploy an app and share it with the community! 🚀

Reviewer 3: ```suggestion
Spaces is a simple and free solution to host Machine Learning demo applications directly on your user profile or your organization [hf.co](http://hf.co/) profile. We support two awesome SDKs that let you build cool apps easily in Python: [Gradio](https://gradio.app/) and [Streamlit](https://streamlit.io/). In a matter of minutes you can deploy an app and share it with the community! 🚀
```

Spaces lets you [set up secrets](/docs/hub/spaces#secret-management), permits [custom requirements](/docs/hub/spaces#how-can-i-install-other-dependencies), and can even be managed [directly from GitHub repos](/docs/hub/spaces#how-can-i-manage-my-app-through-github). You can sign up for the beta at [hf.co/spaces](/spaces). Here are some of our favorites!

- Create recipes with the help of [Chef Transformer](/spaces/flax-community/chef-transformer)
- Transcribe speech to text with [HuBERT](https://huggingface.co/spaces/osanseviero/HUBERT)
- Do segmentation in a video with the [DINO model](/spaces/nateraw/dino-clips)
- Use [Paint Transformer](/spaces/akhaliq/PaintTransformer) to make paintings from a given picture
- Or you can just explore any of the over [100 existing Spaces](/spaces)!

![Landing page of Spaces](assets/27_summer_at_huggingface/spaces_landing.png)

> ### Share some love

Reviewer 3: Capitalise for consistency with the other sections?
```suggestion
### Share Some Love
```

You can now like any model, dataset, or Space on [http://huggingface.co](http://huggingface.co/), meaning you can share some love with the community ❤️. You can also keep an eye on who's liking what by clicking on the likes box 👀. Go ahead and like your own repos, we're not judging 😉.

![Animation giving a like](assets/27_summer_at_huggingface/likes_animation.gif)

### TensorBoard Integration

In late June, we launched a TensorBoard integration for all our models. If there are TensorBoard traces in the repo, an automatic, free TensorBoard instance is launched for you. This works with both public and private repositories and for any library that has TensorBoard traces!

![Image of a TensorBoard Instance](assets/27_summer_at_huggingface/tensorboard.png)

### Metrics

> In July, we added the ability to list evaluation metrics in model repos by adding them to their model card📈. If you add an evaluation metric under the `model_index` section of your model card, it will be displayed proudly in your model repo.

Reviewer 2: ```suggestion
In July, we added the ability to list evaluation metrics in model repos by adding them to their model card📈. If you add an evaluation metric under the `model-index` section of your model card, it will be displayed proudly in your model repo.
```

![Evaluation Metrics](assets/27_summer_at_huggingface/metrics.png)

If that wasn't enough, these metrics will be automatically linked to the corresponding [Papers With Code](https://paperswithcode.com/) leaderboard. That means as soon as you share your model on the Hub, you can compare your results side-by-side with others in the community. 💪

> Check out [this repo](https://huggingface.co/nateraw/vit-base-beans-demo) as an example, paying close attention to `model_index` section of its [model card](https://huggingface.co/nateraw/vit-base-beans-demo/blob/main/README.md#L12-L25) to see how you can do this yourself and find the metrics in Papers with Code [automatically](https://paperswithcode.com/sota/image-classification-on-beans).

Reviewer 2: ```suggestion
Check out [this repo](https://huggingface.co/nateraw/vit-base-beans-demo) as an example, paying close attention to `model-index` section of its [model card](https://huggingface.co/nateraw/vit-base-beans-demo/blob/main/README.md#L12-L25) to see how you can do this yourself and find the metrics in Papers with Code [automatically](https://paperswithcode.com/sota/image-classification-on-beans).
```

### New Widgets

The Hub has 18 widgets that allow users to try out models directly in the browser.

With our latest integrations to Sentence Transformers, we also introduced two new widgets: feature extraction and sentence similarity. 

The latest **audio classification** widget enables many cool use cases: language identification,  [street sound detection](https://huggingface.co/speechbrain/urbansound8k_ecapa) 🚨, [command recognition](https://huggingface.co/speechbrain/google_speech_command_xvector), [speaker identification](https://huggingface.co/speechbrain/spkrec-xvect-voxceleb), and more! You can try this out with `transformers` and `speechbrain` models today! 🔊 (Beware, when you try some of the models, you might need to bark out loud)

> You can try our early demo of [structured data classification](https://huggingface.co/julien-c/wine-quality) with Scikit Learn. And finally, we also introduced new widgets for image-related models: **text to image**, **image classification**, and **object detection**. Try object detection [here](https://huggingface.co/facebook/detr-resnet-50)!

Reviewer 3: ```suggestion
You can try our early demo of [structured data classification](https://huggingface.co/julien-c/wine-quality) with Scikit-learn. And finally, we also introduced new widgets for image-related models: **text to image**, **image classification**, and **object detection**. Try object detection [here](https://huggingface.co/facebook/detr-resnet-50)!
```

> You can try our early demo of [structured data classification](https://huggingface.co/julien-c/wine-quality) with Scikit-learn. And finally, we also introduced new widgets for image-related models: **text to image**, **image classification**, and **object detection**. Try object detection [here](https://huggingface.co/facebook/detr-resnet-50)!

Reviewer 4: As we introduced 3 new image-related widgets, I would add links to each of them (currently there's only a link to the object detection widget)
```suggestion
You can try our early demo of [structured data classification](https://huggingface.co/julien-c/wine-quality) with Scikit-learn. And finally, we also introduced new widgets for image-related models: **text to image**, **image classification**, and **object detection**. Try object detection with Facebook AI's DETR model [here](https://huggingface.co/facebook/detr-resnet-50)!
```

Author: Good suggestion. I'll add a link to the image classification one with ViT, but will skip text to image since that one is only supported in one repo at the moment

![Object Detection Widget](assets/27_summer_at_huggingface/object-detection.png)

### More Features

> That's not everything that has happened in the Hub. We've introduced new and improved [documentation](https://huggingface.co/docs/hub/main) of the Hub. We also introduced two widely requested features: users can now transfer/rename repositories and upload directly new files to the Hub.

Reviewer 4: ```suggestion
That's not everything that has happened in the Hub. We've introduced new and improved [documentation](https://huggingface.co/docs/hub/main) of the Hub. We also introduced two widely requested features: users can now transfer/rename repositories and directly upload new files to the Hub.
```
Sorry, I'm nitpicking

![Button to upload a file](assets/27_summer_at_huggingface/upload_file.png)

## Community

### Hugging Face Course

In June, we launched the first part of our [free online course](https://huggingface.co/course/chapter1)! The course teaches you everything about the 🤗 Ecosystem: Transformers, Tokenizers, Datasets, Accelerate, and the Hub. You can also find links to the course lessons in the official documentation of our libraries. The live sessions for all chapters can be found on our [YouTube channel](https://www.youtube.com/playlist?list=PLo2EIpI_JMQuQ8StH9RwKXwJVqLTDxwwy). Stay tuned for the next part of the course which we'll be launching later this year!

![Course topics](assets/27_summer_at_huggingface/course.png)

### JAX/FLAX Sprint

> In July we hosted our biggest [community event](https://discuss.huggingface.co/t/open-to-the-community-community-week-using-jax-flax-for-nlp-cv/7104) ever with almost 800 participants! In this event co-organized with the JAX/Flax and Google Cloud teams, compute-intensive NLP, Computer Vision, and Speech projects were made practicable for a wider audience of engineers and researchers. The participants created over 170 models, 22 datasets, and 38 Spaces demos 🤯. You can explore all the amazing demos and projects [here](https://huggingface.co/flax-community).

Reviewer 3: ```suggestion
In July we hosted our biggest [community event](https://discuss.huggingface.co/t/open-to-the-community-community-week-using-jax-flax-for-nlp-cv/7104) ever with almost 800 participants! In this event co-organized with the JAX/Flax and Google Cloud teams, compute-intensive NLP, Computer Vision, and Speech projects were made accessible to a wider audience of engineers and researchers. The participants created over 170 models, 22 datasets, and 38 Spaces demos 🤯. You can explore all the amazing demos and projects [here](https://huggingface.co/flax-community).
```

> In July we hosted our biggest [community event](https://discuss.huggingface.co/t/open-to-the-community-community-week-using-jax-flax-for-nlp-cv/7104) ever with almost 800 participants! In this event co-organized with the JAX/Flax and Google Cloud teams, compute-intensive NLP, Computer Vision, and Speech projects were made accessible to a wider audience of engineers and researchers. The participants created over 170 models, 22 datasets, and 38 Spaces demos 🤯. You can explore all the amazing demos and projects [here](https://huggingface.co/flax-community).

Reviewer 4: Perhaps we could mention here that users were offered a free TPUv3, which is pretty amazing

There were talks around JAX/Flax, Transformers, large-scale language modeling, and more! You can find all recordings [here](https://github.com/huggingface/transformers/tree/master/examples/research_projects/jax-projects#talks). 

We're really excited to share the work of the 3 winning teams!

1. [Dall-e mini](https://huggingface.co/spaces/flax-community/dalle-mini). DALL·E mini is a model that generates images from any prompt you give! DALL·E mini is 27 times smaller than the original DALL·E and still has impressive results.

    ![Image generated of an avocado in space](assets/27_summer_at_huggingface/dalle.png)

2. [DietNerf](https://huggingface.co/spaces/flax-community/DietNerf-Demo). DietNerf is a 3D neural view synthesis model designed for few-shot learning of 3D scene reconstruction using 2D views. This is the first Open Source implementation of the "[Putting Nerf on a Diet](https://arxiv.org/abs/2104.00677)" paper.

    ![Generated 3D object with NeRF](assets/27_summer_at_huggingface/diet_nerf.png)

> 3. [CLIP RSIC](https://huggingface.co/spaces/sujitpal/clip-rsicd-demo). CLIP RSIC is a CLIP model fine-tuned on remote sensing image data to enable zero-shot satellite image classification and captioning. This project demonstrates how effective fine-tuned CLIP models can be for specialized domains

Reviewer 2: ```suggestion
3. [CLIP RSIC](https://huggingface.co/spaces/sujitpal/clip-rsicd-demo). CLIP RSIC is a CLIP model fine-tuned on remote sensing image data to enable zero-shot satellite image classification and captioning. This project demonstrates how effective fine-tuned CLIP models can be for specialized domains.
```

    ![CLIP search](assets/27_summer_at_huggingface/clip.png)

> Apart from this very cool projects, we're excited about how these community events enable training large and multi-modal models for multiple languages. For example, we saw the first ever Open Source big LMs for some languages like Swahili, Polish and Marathi.

Reviewer 2: ```suggestion
Apart from this very cool projects, we're excited about how these community events enable training large and multi-modal models for multiple languages. For example, we saw the first ever Open Source big LMs for some languages like [Swahili](https://huggingface.co/models?filter=sw), [Polish](https://huggingface.co/models?filter=pl) and [Marathi](https://huggingface.co/models?filter=mr).
```

Author: Maybe we should instead link directly to the Hub repos since there are non-flax-event models in there

Reviewer 2: oh yeah indeed, forgot it was in the flax event section. As you see fit :)

Reviewer 5: ```suggestion
Apart from these very cool projects, we're excited about how these community events enable training large and multi-modal models for multiple languages. For example, we saw the first ever Open Source big LMs for some low-resource languages like Swahili, Polish and Marathi.
```

Author: Thanks for the suggestions, I added links now

> Apart from this very cool projects, we're excited about how these community events enable training large and multi-modal models for multiple languages. For example, we saw the first ever Open Source big LMs for some languages like Swahili, Polish and Marathi.

Reviewer 3: We can can do without the LM acronym IMO
```suggestion
Apart from these very cool projects, we're excited about how these community events enable training large and multi-modal models for multiple languages. For example, we saw the first ever Open Source big language models for some languages like Swahili, Polish and Marathi.
```

- Hugging Face has collaborated with [NLP in Spanish](https://twitter.com/NLP_en_ES) and [SpainAI](https://twitter.com/Spain_AI_) in a Spanish [course](https://www.youtube.com/playlist?list=PLBILcz47fTtPspj9QDm2E0oHLe1p67tMz) that teaches concepts and state-of-the art architectures as well as their applications through use cases.

## Open Source

### New in Transformers

Summer has been an exciting time for 🤗 Transformers! The library reached 50,000 stars, 30 million total downloads, and almost 1000 contributors! 🤩

So what's new? JAX/Flax is now the 3rd supported framework with over [5000](https://huggingface.co/models?library=jax&sort=downloads) models in the Hub! You can find actively maintained [examples](https://github.com/huggingface/transformers/tree/master/examples/flax) for different tasks such as text classification. We're also working hard on improving our TensorFlow support: all our [examples](https://github.com/huggingface/transformers/tree/master/examples/tensorflow) have been reworked to be more robust, TensorFlow idiomatic, and clearer. This includes examples such as summarization, translation, and named entity recognition.

> You can now easily publish your model to the Hub, including automatically authored model cards, evaluation metrics, and TensorBoard instances. There is also increased support for exporting models to ONNX with the new `transformers.onnx` library.

Reviewer 3: Is `transformers.onnx` a library or a module?

Author: Good point, changed to module!

> You can now easily publish your model to the Hub, including automatically authored model cards, evaluation metrics, and TensorBoard instances. There is also increased support for exporting models to ONNX with the new `transformers.onnx` module.

Reviewer 4: Here, we could 
```suggestion
You can now easily publish your model to the Hub, including automatically authored model cards, evaluation metrics, and TensorBoard instances. There is also increased support for exporting models to ONNX with the new [`transformers.onnx` module](https://huggingface.co/transformers/serialization.html?highlight=onnx). 
```

```bash
python -m transformers.onnx --model=bert-base-cased onnx/bert-base-cased/
```

The last 4 releases introduced many new cool models!

- DETR ([paper](https://arxiv.org/abs/2005.12872)) can do fast end-to-end object detection and image segmentation. Check out some of our community [tutorials](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DETR)!

![DETR image](assets/27_summer_at_huggingface/detr.png)

> - ByT5 ([paper](https://arxiv.org/abs/2105.13626)) is the first tokenizer-free model in the Hub! You can find all available checkpoints [here](https://huggingface.co/models?filter=arxiv:1907.06292).

Reviewer 4: I've also added CANINE, which is another tokenizer-free model, so I would add it here. Also, rather than filtering on the arxiv link of ByT5 on the hub, there are actually already a lot more checkpoints available if you just filter on "byt5".
```suggestion
- ByT5 ([paper](https://arxiv.org/abs/2105.13626)) is the first tokenizer-free model in the Hub! You can find all available checkpoints [here](https://huggingface.co/models?search=byt5).
- CANINE ([paper](https://arxiv.org/abs/2103.06874)) is another tokenizer-free encoder-only model by Google AI, operating directly at the character level. You can find all (multilingual) checkpoints [here](https://huggingface.co/models?search=canine).
```

- HuBERT ([paper](https://arxiv.org/abs/2106.07447)) shows exciting results for downstream audio tasks such as [command classification](https://huggingface.co/superb/hubert-base-superb-ks) and [emotion recognition](https://huggingface.co/superb/hubert-base-superb-er). Check the models [here](https://huggingface.co/models?filter=hubert).

> - LayoutLM-v2 ([paper](https://arxiv.org/abs/2012.14740)) and LayoutXLM are two incredible models capable of parsing document images (like PDFs) by incorporating text, layout, and visual information. We built a [Space demo](https://huggingface.co/spaces/nielsr/LayoutLMv2-FUNSD) so you can directly try it out!

Reviewer 4: ```suggestion
- LayoutLMv2 ([paper](https://arxiv.org/abs/2012.14740)) and LayoutXLM ([paper](https://arxiv.org/abs/2104.08836)) are two incredible models capable of parsing document images (like PDFs) by incorporating text, layout, and visual information. We built a [Space demo](https://huggingface.co/spaces/nielsr/LayoutLMv2-FUNSD) so you can directly try it out! Demo notebooks can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/LayoutLMv2).
```

> ![LayotLM object detection](assets/27_summer_at_huggingface/layout.png)

Reviewer 4: ```suggestion
![LayoutLM object detection](assets/27_summer_at_huggingface/layout.png)
```

> - BEiT ([paper](https://arxiv.org/abs/2106.08254)) makes self-supervised pretraining of Vision Transformers outperform supervised pre-training.

Reviewer 4: ```suggestion
- BEiT ([paper](https://arxiv.org/abs/2106.08254)) by Microsoft Research makes self-supervised pretraining of Vision Transformers outperform supervised pre-training, using a pre-training objective inspired by BERT.
```

Author: Good idea about linking to docs instead of papers, I'll follow-up with a commit for that

The Hub is now integrated into `transformers`, with the ability to push to the Hub configuration, model, and tokenizer files without leaving the Python runtime! The `Trainer` can now push directly to the Hub every time a checkpoint is saved: 

![Untitled](assets/27_summer_at_huggingface/save_checkpoint.png)

> ### New in Datasets

Reviewer 3: Should we mention the awesome new docs for `datasets`? For example,

> The library also has fancy new documentation - check it out [here](https://huggingface.co/docs/datasets/index.html)!

Author: Originally this post was only intended for things that happened before September but people have added new stuff. If possible I would prefer not to add any new content, but if @lhoestq or @stevhliu would like to add few sentences about it, it would be great :)

Users can also directly host and share their datasets to the community simply by uploading their data files in a repository on the Dataset Hub.

![Untitled](assets/27_summer_at_huggingface/streaming.png)

What are the new datasets highlights? Microsoft CodeXGlue [datasets](https://huggingface.co/datasets?search=code_x_glue) for multiple coding tasks (code completion, generation, search, etc), huge datasets such as [C4](https://huggingface.co/datasets/c4) and [MC4](https://huggingface.co/datasets/mc4), and many more such as  [RussianSuperGLUE](https://huggingface.co/datasets/russian_super_glue) and [DISFL-QA](https://huggingface.co/datasets/disfl_qa).

### Third Party Integrations to the Hub

Apart from having deep integration with `transformers`-based models, the Hub is also building great partnerships with open source ML libraries to provide free model hosting and versioning. We've been achieving this with our [huggingface_hub](https://github.com/huggingface/huggingface_hub) Open-Source library as well as new Hub [documentation](https://huggingface.co/docs/hub/main). 

> All spaCy canonical pipelines can now be found in the official spaCy [organization](https://huggingface.co/spacy), and any user can share their pipelines with a single command `python -m spacy huggingface-hub`. To read more about it, head to [https://huggingface.co/blog/spacy](https://huggingface.co/blog/spacy). You can try all canonical spaCy models directly in the Hub in the demo [Space](https://huggingface.co/spaces/spacy/pipeline-visualizer)!

Reviewer 2: fyi I get an error when launching the space (didn't touch anything):

<img width="886" alt="image" src="https://user-images.githubusercontent.com/5020707/134230601-8d87d330-22c4-4ed8-9aa0-e8175eb990c8.png">

(cc @cbensimon?)

Author: Yes, this and the DietNerf Spaces are unfortunately broken at the moment. The `spacy` one seems to be related to the version but I'm not sure

Author: Space is fixed now

### **NEW: Inference on SageMaker**

We launched a [new integration with AWS](https://huggingface.co/blog/deploy-hugging-face-models-easily-with-amazon-sagemaker) to make it easier than ever to deploy 🤗 Transformers in SageMaker 🔥. Pick up the code snippet right from the 🤗 Hub model page! Learn more about how to leverage transformers in SageMaker in our [docs](https://huggingface.co/docs/sagemaker/inference) or check out these [video tutorials](https://youtube.com/playlist?list=PLo2EIpI_JMQtPhGR5Eo2Ab0_Vb89XfhDJ).

For questions reach out to us on the forum: [https://discuss.huggingface.co/c/sagemaker/17](https://discuss.huggingface.co/c/sagemaker/17)

![Sagemaker](assets/27_summer_at_huggingface/sagemaker.png)

> ### **NEW: AutoNLP in your browser**

Reviewer 3: ```suggestion
### **NEW: AutoNLP In Your Browser**
```

**Hugging Face** + **Zapier Demo**

20,000+ Machine Learning models connected to 3,000+ apps? 🤯  By leveraging the [Inference API](https://huggingface.co/landing/inference-api/startups), you can now easily connect models right into apps like Gmail, Slack, Twitter, and more. In this demo video, we created a zap that uses this [code snippet](https://gist.github.com/feconroses/3476a91dc524fdb930a726b3894a1d08) to analyze your Twitter mentions and alerts you on Slack about the negative ones.

<iframe width="700" height="394" src="https://www.youtube.com/embed/sjfpOJ4KA78" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

**Hugging Face + Google Sheets Demo**

With the [Inference API](https://huggingface.co/landing/inference-api/startups), you can easily use zero-shot classification right into your spreadsheets in Google Sheets. Just [add this script](https://gist.github.com/feconroses/302474ddd3f3c466dc069ecf16bb09d7) in Tools -> Script Editor:

> <iframe width="700" height="394" src="https://www.youtube.com/embed/-A-X3aUYkDs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Reviewer 4: Should we link to the table question answering demo in Google Sheets here too? It got quite a lot of interest both on Twitter/Linkedin

Transformers latency down to 1ms? 🤯🤯🤯

Hugging Face is working on a really special solution to achieve unmatched performance of state-of-the-art Transformer models for companies to deploy in their own infrastructure.

- Infinity comes as a single-container and can be deployed in any production environment.
- It can achieve 1ms latency for BERT-like models on GPU and 4-10ms on CPU 🤯🤯🤯
- Infinity meets the highest security requirements and can be integrated into your system without the need for internet access. You have control over all incoming and outgoing traffic.

⚠️  Join us for a [live announcement and demo on Sep 28](https://app.livestorm.co/hugging-face/hugging-face-infinity-launch?type=detailed), where we will be showcasing Infinity for the first time in public!

## Research

> At BigScience we held our first live event (since the kick off) in July BigScience Episode #1. Our second event BigScience Episode #2 was held on September 20th, 2021 with technical talks and updates by the BigScience working groups and invited talks by Jade Abbott (Masakhane), Percy Liang (Stanford CRFM), Stella Biderman (EleutherAI) and more. We have completed the first large-scale training on Jean Zay, a 13B English only decoder model (you can find the details here), and we're currently deciding on the architecture of the second model. The organization working group has filed the application for the second half of the compute budget: Jean Zay V100 : 2,500,000 GPU hours. 🚀 

Reviewer 6: `(you can find details here)` -> i think there is a link missing...

Author: Good catch! Fixed
